### ISSUE TITLE
Language code conflict

### ISSUE THREAD

**Issue by BrutuZ** [rockrider69/service.LanguagePreferenceManager#9]
Language code `pt-br` conflicts with the subtag delimiter for Signs & Songs (`ss`), making it unable to set the language on the Custom Condition fields as it is split into `pt` and `br` 

### Setting:
```xml
    <setting id="CustomCondSub">jpn:pob&gt;jpn:por&gt;jpn:eng&gt;jpn:any&gt;eng:pob&gt;eng:por&gt;eng:jpn&gt;eng:eng&gt;pob:non&gt;por:non</setting>
```
### Log:
```
 info <general>: [Language Preference Manager]: Custom cond subs prefs parse error: ['jpn', 'pt-br']. Unknown sub tag is ignored                      
 info <general>: [Language Preference Manager]: Custom cond sub prefs: lang code not found in db! Please report this: ('Japanese', 'jpn'):(None, 'pt')
 info <general>: [Language Preference Manager]: Custom cond subs prefs parse error: ['eng', 'pt-br']. Unknown sub tag is ignored                      
 info <general>: [Language Preference Manager]: Custom cond sub prefs: lang code not found in db! Please report this: ('English', 'eng'):(None, 'pt') 
```

### References:

https://github.com/rockrider69/service.LanguagePreferenceManager/blob/33870587bc4c11e2114ed51b2b090ac704acd470/resources/lib/langcodes.py#L45

https://github.com/rockrider69/service.LanguagePreferenceManager/blob/33870587bc4c11e2114ed51b2b090ac704acd470/resources/lib/prefparser.py#L34

https://github.com/rockrider69/service.LanguagePreferenceManager/blob/33870587bc4c11e2114ed51b2b090ac704acd470/resources/lib/prefparser.py#L83-L94

**Comment by rockrider69** [comment]
Thank you for reporting this and the clean analysis ;-)
Let me see how to fix it the right way and come back.

**Comment by rockrider69** [comment]
Hello !
thank you for your fix proposal, I am fine with it of course !
Your large pull request from my master should work (although I forked mine last year from fpatrick who was already ahead of ace2002...).
Maybe to make things cleaner, it is better for me to first merge the current 1.0.3 branch into my own master, and apply your very last commit. And later on anyone can propose real small PRs from there.

What do you think ? I am not so expert GitHub user...

**Comment by BrutuZ** [comment]
This PR was based on the V1.0.3 branch, so merging it to master will have the same effect as pulling the branch (just don't squash it)

**Comment by BrutuZ** [comment]
You can also create a new V1.0.4 branch based on V1.0.3 and I'll change the PR target to that, then you decide whether or not to update the master branch and when

**Comment by rockrider69** [comment]
OK I just created a new V1.0.4 branch from V1.0.3 (which is the current official version in KODI Repository). You can propose any PRs from this one.
Appreciate the help and advices, I am still a bit of a rookie python ;)

**Comment by rockrider69** [comment]
Merged into new V1.0.4 branch.
